.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature Request
+description: A suggestion for a new feature
+labels: [enhancement]
```

Comment:
I think there's an explicit new feature label if that is what this is pulling from.